<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>951</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(170, 170, 127)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Sitka Banner&quot;;</string>
    </property>
    <property name="text">
     <string>Device Tracking System</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="image">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="imageButton">
       <property name="text">
        <string>Select Image</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>520</y>
      <width>217</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>122</y>
      <width>418</width>
      <height>268</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>25</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>30</number>
       </property>
       <item>
        <widget class="QLabel" name="deviceName">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Device Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="deviceNameT"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>24</number>
       </property>
       <item>
        <widget class="QLabel" name="serialNo">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Serial Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="serialNoT">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>60</number>
       </property>
       <item>
        <widget class="QLabel" name="assign">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Assigned</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="assignT"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>351</width>
      <height>53</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>15</number>
     </property>
     <property name="rightMargin">
      <number>40</number>
     </property>
     <item>
      <widget class="QLabel" name="location">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Location</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="styleSheet">
        <string notr="true">padding : 10px,20px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>270</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>pdfGenerator</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
